src/convertToObject.js

```diff
@@ -12,7 +12,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrOfProps = sourceString.split(';')
+    .map(prop => prop.split(':').map(x => x.trim()));
+
+  const newObj = {};
```

Comment:
недескриптивна назва
```suggestion
  const styles = {};
```